bin/package

```diff
@@ -1,55 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
This is the part that builds the package for releases. We shouldn't delete it until we have a replacement.

Reply:
You mean it builds all the PHARs right?